db/schema.rb

```diff
@@ -103,6 +103,7 @@
     t.datetime "created_at",                          null: false
     t.datetime "updated_at",                          null: false
     t.boolean  "with_us",             default: false
+    t.string   "twitter_id"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.